readthedocs/templates/docsitalia/come-pubblicare.html

```diff
@@ -70,7 +72,14 @@ <h3 class="mt-5 pt-4" id="lista-articolo-4">4. Converti il tuo documento</h3>
             <p>Puoi scrivere i documenti direttamente in RST, oppure se il tuo documento è già in un altro formato (es. ODT, DOCX, o Markdown) puoi usare questo convertitore:</p>
             <div class="my-3 text-center">
               {% url 'docs_italia_convertitore_web:docs-italia-converter-view' as converti_url %}
-              <a class="btn btn-sm btn-outline-primary btn-outline-primary-white m-3 m-lg-0" href="{{ converti_url }}" aria-label="Converti" title="Converti"><span class="text-uppercase small">Converti un documento<i class="it-upload pl-2"></i></span></a>
+              <a class="btn btn-sm btn-icon btn-outline-primary btn-outline-primary-white m-3 m-lg-0" href="{{ converti_url }}" aria-label="Converti" title="Converti">
```

Comment:
Riguardo questo (che è corretto), ho notato che in locale non è possibile verificare l'url `converti` (a me torna un 404) perché ha un processo di build separato, se non ricordo male? Ne parliamo all'allineamento comunque.

Reply:
@francescozaia ci fabio abbiamo trovato un problema con il convertitore, è in arrivo PR per sistemarlo